src/find-specs.js

```diff
@@ -0,0 +1,74 @@
+const fetch = require("node-fetch");
+
+const specs = require("../index.json");
+
+const nonBrowserSpecWgs = ["Accessibility Guidelines Working Group", "Accessible Platform Architectures Working Group", "Automotive Working Group", "Dataset Exchange Working Group", "Decentralized Identifier Working Group", "Distributed Tracing Working Group", "Education and Outreach Working Group", "JSON-LD Working Group", "Publishing Working Group", "Verifiable Credentials Working Group", "Web of Things Working Group"];
```

Comment:
Can you put one on each line and sort to make it easier to read/update this? Likewise below.